app/views/state_file/questions/income_review/edit.html.erb

```diff
@@ -81,10 +85,15 @@
   <% end %>
 
   <% if current_intake.direct_file_data.fed_ssb > 0 || current_intake.direct_file_data.fed_taxable_ssb > 0 %>
-    <div class="white-group">
+    <div class="white-group" id="formssa1099s">
       <p class="text--bold spacing-below-0"><%= t(".ssa_title") %></p>
       <p class="text--grey-bold spacing-above-15 spacing-below-0">
-        <%= t(".no_info_needed") %>
+        <%= case current_state_code
+        when "nj"
+          t(".nj_not_taxed")
+        else
+          t(".no_info_needed")
+        end %>
```

Comment:
[dust] I think we generally have been trying to steer away from checking `current_state_code` to enter in state-specific content. Ways we've been trying to avoid it:
1. adding flags on the `StateFileBaseIntake` and `StateFile[state]Intake`  
2. creating partials
3. creating state-specific translations

For this above scenario, is it possible to do something like below? (from `app/views/state_file/questions/return_status/_accepted.html.erb`:

```
      <% # i18n-tasks-use t("state_file.questions.return_status.accepted.check_payable_html") # hint for the i18n linter that, yes, we are using this key (sometimes) %>
      <%= t(".#{current_state_code}.check_payable_html", default: :'.check_payable_html', payable_to: @department_of_taxation, filing_year: current_tax_year, form_number: @tax_form_number) %>
```

we have a `default` (`.no_info_needed`) here and state-specific copy (`.nj_not_taxed` -> `.nj.no_info_needed`)?

Reply:
thanks -- that is a great tip about the linter hint; i definitely wrestled a bit with that on this. i'll update to use your suggestion to not check state code there...

Reply:
used the linter hint and default setup to avoid checking for "nj" specifically. thanks!